apps/api/src/test-data/buildBibliographicReferenceTestData/buildJournalArticleBibliographicReferenceTestData.ts

```diff
@@ -20,7 +20,8 @@ const dtos: DTO<JournalArticleBibliographicReference>[] = [
             abstract: 'An analysis of the Cariboo Chilcotin Justice inquiry.',
             issueDate: 'Spring 2013',
             publicationTitle: 'Journal of History',
-            url: 'https://search.proquest.com/docview/1682229477/abstract/7836BCEA06014582PQ/1',
+            url: 'https://coscrad.org/wp-content/uploads/2023/05/Test-Journal-Article-Sample-PDF.pdf',
```

Comment:
Nice work on the test PDFs!

Reply:
Yea, but they aren't the right length. If and when this matters, we can make the dummy media even more realistic.